x-pack/plugins/apm/public/components/app/settings/apm_indices/index.tsx

```diff
@@ -33,13 +33,6 @@ import {
 } from '../../../../services/rest/create_call_apm_api';
 
 const APM_INDEX_LABELS = [
-  {
-    configurationName: 'sourcemap',
-    label: i18n.translate(
-      'xpack.apm.settings.apmIndices.sourcemapIndicesLabel',
-      { defaultMessage: 'Sourcemap Indices' }
-    ),
-  },
```

Comment:
The source map index is no longer configurable (since years ago) and shouldn't be displayed in the UI.